apps/examples/elf/micomapp/Makefile

```diff
@@ -101,9 +105,11 @@ $(BIN): $(OBJS)
 
 clean:
 	$(call DELFILE, $(BIN))
-	$(call DELFILE, $(SYSTEM_BIN_DIR)/$(BIN))
+	$(call DELFILE, $(USER_BIN_DIR)/$(BIN))
 	$(call CLEAN)
 
 install:
-	$(Q) mkdir -p $(SYSTEM_BIN_DIR)
-	$(Q) install $(BIN) $(SYSTEM_BIN_DIR)/$(BIN)
+	$(Q) mkdir -p $(USER_BIN_DIR)
+	$(Q) install $(BIN) $(USER_BIN_DIR)/$(BIN)
+	$(Q) $(TOPDIR)/tools/mkheader.py $(USER_BIN_DIR)/$(BIN) $(BIN_TYPE) $(KERNEL_VER) $(BIN) $(BIN_VER)
```

Comment:
@YiHyunJin How about using `-x` option to say what it is?
Currently, the mkheader.py gets arguments without any pre option. that's why user must know the sequence of them.
I mean
```
mkheader.py -v 19920412 -b micom -t ELF -kv 2.0
```

Reply:
okay. thanks. I'll think about this.